addons/account/models/account_move.py

```diff
@@ -748,10 +748,12 @@ def _compute_base_line_taxes(base_line):
                     'amount': 0.0,
                     'tax_base_amount': 0.0,
                     'grouping_dict': False,
+                    'partner_id': False,
                 })
                 taxes_map_entry['amount'] += tax_vals['amount']
                 taxes_map_entry['tax_base_amount'] += self._get_base_amount_to_display(tax_vals['base'], tax_repartition_line, tax_vals['group'])
                 taxes_map_entry['grouping_dict'] = grouping_dict
+                taxes_map_entry['partner_id'] = line.partner_id
```

Comment:
`partner_id` should be part of the grouping key I think: `_get_tax_grouping_key_from_tax_line` / `_get_tax_grouping_key_from_base_line`

Reply:
:+1: